be/src/olap/olap_cond.cpp

```diff
@@ -656,7 +654,7 @@ int Conditions::delete_pruning_filter(
         }
     }
 
-    if (true == del_not_satisfied || 0 == _columns.size()) {
+    if (del_not_satisfied || _columns.empty()) {
```

Comment:
put _columns.empty() in the first line of this function

Reply:
OK